.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea
```

Comment:
Do we need this on the top level ? Maybe put in sdk dir or sdk/go ?

Reply:
I believe I just edited the one we already have. But yeah, sure, I can add one on `sdk/go` too.